src/site/apt/license_def.apt.vm

```diff
@@ -0,0 +1,143 @@
+ is a Matcher type.  A matcher element is defined by a builder in the "matchers" section as described
+ above.  The matchers listed in the example above define the "any", "spdx", and "text" matcher elements.
+ 
+ When defining the license the matcher is required and only one may be defined.  However, there are matchers
+ that accept multiple matcher child nodes.  In the example below the CDDL1 license uses the "any" matcher.
+ The "any" matcher accepts multiple child matchers and will report a match if any of the enclosed matchers
+ report a match. In the example below the "any" matcher has "text" and "spdx" child matchers.  The "text"
+ matcher will match the enclosed text and the "spdx" matcher matches {{{https://spdx.org/licenses/}SPDX}}
+ tags.  The second license defined below is the "ILLUMOS" license.  It shares the same family as the CDDL1
+ license, has its own "id" and "name" properties.  It also has a "note" child element that tells the user
+ that it is a modified CDDL1 license.  The license has a single "text" matcher.
+ 
++------------------------------------------+
+ <rat-config>
+    <licenses>
+        <license family="CDDL1">
+            <any>
+                <text>The contents of this file are subject to the terms of the
+                    Common Development and Distribution License("CDDL") (the
+                    "License"). You may not use this file except in compliance
+                    with the License.</text>
+                <spdx name='CDDL-1.0' />
+            </any>
+        </license>
+
+        <license family="CDDL1" id="ILLUMOS" name="ILLUMOS CDDL1 Derived license">
+            <note>Modified CDDL1 license</note>
+            <text>The contents of this file are subject to the terms of
+                the Common Development and Distribution License (the
+                "License") You may not use this file except in compliance
+                with the License. </text>
+        </license>
+    </licenses>
+ </rat-config>
++------------------------------------------+
+ 
+* Approved 
+ 
+ Approved element lists the approved licenses from this configuration.  It is possible to define licenses
+ that you want to detect but not allow.  If the "approved" element is not present all defined families
+ are considered to be approved.  The approved entry has "family" child entries that have "license_ref"
+ properties that reference the "id" of the family being approved.
+ 
++------------------------------------------+
+ <rat-config>
+    <approved>
+        <family license_ref='AL' />
+    </approved>
+ </rat-config>
++------------------------------------------+
+
+* Listing components
+
+ All the components (Licenses and Matchers) defined in the system can be displayed using the Documentation
+ tool.  To run the too execute:
+
++------------------------------------------+
+java -jar apache-rat-${project.version}.jar org.apache.rat.Documentation
```

Comment:
Would you mind using the same kind of linking as in 
https://creadur.apache.org/rat/apache-rat/index.html
"apache-rat/target/" ?